tmexp/utils.py

```diff
@@ -1,9 +1,10 @@
+from collections.abc import Mapping as MappingABC, MutableMapping as MutableMappingABC
 import logging
 from logging import Handler, Logger, LogRecord, NOTSET
 import os
 import shutil
 import time
-from typing import List, Optional
```

Comment:
When typing Counters we rename to CounterType, I feel we should do MappingTypy and MutableMappingType as well